File: problems/solutions/3753-maximum-difference-between-even-and-odd-frequency-i/solution.py
class Solution:
    def maxDifference(self, s: str) -> int:
        d = {}
        par, impar = 0, 0
        for i in s:
            if i not in d:
                d[i] = 1
            else:
                d[i] += 1

        for tmp in d.values():
            if tmp % 2:
                if not impar or tmp > impar:
                    impar = tmp

            else:
                if not par or tmp < par:
                    par = tmp
        return impar - par

File: problems/solutions/3753-maximum-difference-between-even-and-odd-frequency-i/solution.php
<?php

class Solution {

    /**
     * @param String $s
     * @return Integer
     */
// 3442. Maximum Difference Between Even and Odd Frequency

    function maxDifference($s) {
        $arr = [];
        [$par, $impar] = [0, 0];

        for ($i = 0; $i < strlen($s); $i++){
            $char = $s[$i];
            if (isset($arr[$char])){
                $arr[$char] += 1;
            }else{
                $arr[$char] = 1;
            }
        }

        foreach ($arr as $_ => $v){
            if (!($v % 2)){
                if (!$par || $v < $par){
                    $par = $v;
                }
            }else {
                if (!$impar || $v > $impar){
                    $impar = $v;
                }
            }
        }
        return $impar - $par;
    }
}
